Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -490,7 +591,7 @@ WarpX::ComputeOneWayExtensions ()
                     return 0;
                 }
 
-                const amrex::Real S_stab = ComputeSStab(i, j, k, lx, ly, lz, dx, dy, dz, idim);
+                const amrex::Real S_stab = ::ComputeSStab(i, j, k, lx, ly, lz, dx, dy, dz, idim);
```

Comment:
Why is it necessary to call the wrapper that takes `idim` as a non-template parameter here (and in similar calls below), instead of calling the new templated function?
```cpp
const amrex::Real S_stab = ::ComputeSStab<idim>(i, j, k, lx, ly, lz, dx, dy, dz);
```

Reply:
@EZoni , I don't think that we can do that here. This instruction is inside a for loop: `for(int idim = 0; idim < AMREX_SPACEDIM; ++idim)` .

Reply:
ping @EZoni  ;-)